listenbrainz/webserver/static/js/src/personal-recommendations/PersonalRecommendationsModal.tsx

```diff
@@ -0,0 +1,258 @@
+        id="PersonalRecommendationModal"
+        tabIndex={-1}
+        role="dialog"
+        aria-labelledby="PersonalRecommendationModalLabel"
+        data-backdrop="static"
+      >
+        <div className="modal-dialog" role="document">
+          <form className="modal-content">
+            <div className="modal-header">
+              <button
+                type="button"
+                className="close"
+                data-dismiss="modal"
+                aria-label="Close"
+                onClick={this.closeModal}
+              >
+                <span aria-hidden="true">&times;</span>
+              </button>
+              <h4 className="modal-title" id="PersonalRecommendationModalLabel">
+                Personally recommend this recording
+              </h4>
+            </div>
+            <div className="modal-body">
+              {users.map((user) => {
+                return (
+                  <NamePill
+                    title={user}
+                    // eslint-disable-next-line react/jsx-no-bind
+                    closeAction={this.removeUser.bind(this, user)}
+                  />
+                );
+              })}
+              <input
+                type="text"
+                className="form-control"
+                onChange={this.searchUsers}
+                placeholder="Search and add users"
+              />
+              <SearchDropDown
+                suggestions={suggestions}
+                // eslint-disable-next-line react/jsx-no-bind
+                action={this.addUser}
+              />
+              <p>
+                Tell the users selected above why are you recommending them{" "}
+                <b>
+                  {" "}
+                  {track_name} by {artist_name}
+                </b>
+                (Optional)
+              </p>
+              <div className="form-group">
+                <textarea
+                  className="form-control"
+                  id="blurb-content"
+                  placeholder="Let your followers know why you recommended this song"
+                  value={blurbContent}
+                  name="blurb-content"
+                  rows={4}
+                  style={{ resize: "vertical" }}
+                  onChange={this.handleBlurbInputChange}
+                />
+              </div>
+              <small>
+                {blurbContent.length} / {this.maxBlurbContentLength}
+              </small>
+            </div>
+            <div className="modal-footer">
+              <button
+                type="button"
+                className="btn btn-default"
+                data-dismiss="modal"
+              >
+                Cancel
+              </button>
+              <button
+                type="submit"
+                className="btn btn-success"
+                data-dismiss="modal"
+                onClick={this.submitPersonalRecommendation}
+              >
+                Personally Recommend
```

Comment:
I'm on the fence with the wording (both here and on the listencard dropdown menu item).
What do you think of "Send to a friend" or "Send to a follower"?
Or if we want to stay with "recommend", "Recommend to a follower"?

You could ask around on the IRC channel, perhaps someone else has a good idea.